src/utils/jwt.js

```diff
@@ -27,4 +27,12 @@ export default class jwtHelper {
     const token = jwt.sign(userId, secret, { expiresIn: '1d' });
     return token;
   }
+
+  // authorization for admin
+  static async verifyAdmin(user) {
```

Comment:
@bright258 this should also be a middleware, it will strictly be for admin authorization, and this function is not rightt

Reply:
Okay. I am working on it

Reply:
I have fixed it